remotes/docker/config/hosts.go

```diff
@@ -322,6 +337,12 @@ type hostFileConfig struct {
 	OverridePath bool `toml:"override_path"`
 
 	// TODO: Credentials: helper? name? username? alternate domain? token?
+
+	// Username is the username used to authenticate to the server
+	Username string `toml:"username"`
+
+	// Password is the password used to authenticaate to the server
+	Password string `toml:"password"`
```

Comment:
We do need to discuss about the direction first. I don't love to have passwords in plaintext.